jwst/master_background/master_background_step.py

```diff
@@ -17,9 +16,9 @@ class MasterBackgroundStep(Step):
     spec = """
         user_background = string(default=None) # Path to user-supplied master background
         save_background = boolean(default=False) # Save computed master background
+        subtract_background = boolean(default=None) #Subtract master background
```

Comment:
space after the `#`

Reply:
Also, how will this interact with `skip`, which is built-in to the step?

Reply:
Hmmm, @jdavies-st that's a good question. The normal way a user requests skipping is via the step class "skip" param. There's no equivalent for them to force a step to execute (just the absence of step.skip). In the `extract_1d` step it made sense to have a separate `subtract_background` param, because the step is still doing other work. But in this case we're trying to turn the entire step on or off. Perhaps we need to rethink this strategy a bit.